tools/include/eosio/gen.hpp

```diff
@@ -534,7 +534,7 @@ struct generation_utils {
    }
 
    void translate_explicit_nested_map_or_pair(const clang::QualType& type, int depth, std::string & ret, const std::string & tname, bool & gottype){
-      ret += depth > 0 ? tname + "_" : "pair_";
+      ret += depth > 0 ? tname + "_" : tname == "map" ? "kvpair_" : "pair_";
```

Comment:
The purpose is to distinguish map and pair, I suggest change "kvpair_" to "map" when tname=="map", because 'kvpair' is kind of misleading, people might think 'kvpair' is only related to kv-table or kv-map

Reply:
As map_A_B[] would looks more like vector<map>, after talk with Ping, we agree with using mpair_A_B to represent a element of map, so kvpair will change to mpair.